toolkit/Makefile

```diff
@@ -92,6 +92,7 @@ CCACHE_DIR       ?= $(PROJECT_ROOT)/ccache
 LOGS_DIR           ?= $(BUILD_DIR)/logs
 PKGBUILD_DIR       ?= $(BUILD_DIR)/pkg_artifacts
 CACHED_RPMS_DIR    ?= $(BUILD_DIR)/rpm_cache
+REPO_QUERY_DIR     ?= $(BUILD_DIR)/repo-query
```

Comment:
I'd suggest `repo_query` to match the other directories which use underscore instead of a dash

Reply:
thx - fixed.